appservice/src/deploy/localGitDeploy.ts

```diff
@@ -35,6 +35,8 @@ export async function localGitDeploy(client: SiteClient, fsPath: string): Promis
         // tslint:disable-next-line:no-floating-promises
         localGit.push(remote, 'HEAD:master');
     } catch (err) {
+        const parsedError: IParsedError = parseError(err);
+        parsedError.message.replace(new RegExp(nonNullProp(publishCredentials, 'publishingPassword'), 'g'), '***');
```

Comment:
I think string.replace is non-destructive...?

Reply:
You are absolutely correct.  I need to re-assign 😃

Reply:
UNIT TESTS